src/components/atoms/TeamCard/index.jsx

```diff
@@ -0,0 +1,42 @@
+import TwitterIcon from '@mui/icons-material/Twitter';
+import Body3 from '../../shared/Typography/Body3';
+import Heading1 from '../../shared/Typography/Heading1';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import { MeetTheTeam, Socials, TeamCards, Wrapper, Heading } from './styles';
+import { Links } from './styles';
+import Heading4 from '../../shared/Typography/Heading4';
+import TeamData from '../../../data/TeamCardData/TeamData.json';
+
+const TeamCard = () => {
+  return (
+    <MeetTheTeam>
+      <Heading>
+        <Heading1>Meet The Team</Heading1>
+        <Heading4>Get to Know the talented individuals behind our club</Heading4>
+      </Heading>
+      <TeamCards>
+        {TeamData.map(TeamMember => (
+          <Wrapper key={TeamMember}>
```

Comment:
Make a team card and put it in atom. And this component should be TeamSection.

Reply:
Done